CHANGELOG.md

```diff
@@ -1,7 +1,196 @@
 # Changelog
 
-> All notable changes to this project are documented in this file.
-> This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+> All notable changes to this project are documented in this file. This project
+> adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [[v3.0.0]](https://github.com/springload/react-accessible-accordion/releases/tag/v3.0.0)
+
+This release is the culmination of a massive amount of work, resulting in some
+new features and significantly stronger and more reliable WAI ARIA spec
+compliance. Notably, the project has been migrated from Flow to Typescript, and
+a full accessibility audit has been performed - revealing a few holes in our
+compliance which have now been entirely addressed.
+
+Thanks to everyone who has contributed to this release - and not just those who
+have written code. Contribution by way of issues relating to spec compliance,
+pull-request commentary, advice and assistance is all greatly appreciated.
+Thanks also to the patient users who have endured an extended period without a
+release while we made sure to get this 100% right! Release cadence should return
+to normal again now.
+
+### Breaking Changes - Upgrade Guide:
+
+1. Rename all of your `AccordionItemTitle` components to `AccordionItemHeading`
+   and then nest an `AccordionItemButton` directly inside of each one. Note that
+   in order for your Accordion to remain spec-compliant, you may _not_ put any
+   other children inside `AccordionItemHeading`.
+
+    ```tsx
+    // Before:
+    import { AccordionItemTitle } from 'react-accessible-accordion';
+
+    const headingBefore = <AccordionItemHeading>Foo</AccordionItemHeading>;
```

Comment:
Should this be `const headingBefore = <AccordionItemTitle>Foo</AccordionItemTitle>;`?

Reply:
Yep, nice catch.